ios/FluentUI/Navigation/BadgeLabelButton.swift

```diff
@@ -208,5 +208,13 @@ class BadgeLabelButton: UIButton {
 
     @objc private func badgeValueDidChange() {
         updateBadgeLabel()
+        updateAccessibilityLabel()
+    }
+
+    private func updateAccessibilityLabel() {
+        guard let item = item else {
+            return
+        }
+        accessibilityLabel = (item.accessibilityLabel ?? "").appending(item.badgeAccessibilityLabel ?? "")
```

Comment:
We cannot just append strings like this because of localization scenarios.
A string like "%@, %@" needs to be created so that the localization team can arrange the order and/or eventually replace the comma with any other character of the specific language.

Reply:
Also, the localized string for concatenating both accessibility labels will be used only if the badgeAccessibilityLabel is provided.

Reply:
Done!!